src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6073,4 +6073,25 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureModuleInitializers" xml:space="preserve">
     <value>module initializers</value>
   </data>
+  <data name="ERR_ModuleInitializerMethodMustBeAccessibleOutsideTopLevelType" xml:space="preserve">
+    <value>Module initializer method '{0}' must be accessible outside top-level type '{1}'</value>
+  </data>
+  <data name="ERR_ModuleInitializerMethodMustBeStatic" xml:space="preserve">
+    <value>Module initializer method '{0}' must be static</value>
+  </data>
+  <data name="ERR_ModuleInitializerMethodMustNotBeContainedInGenericType" xml:space="preserve">
+    <value>Module initializer method '{0}' must not be contained in a generic type</value>
+  </data>
+  <data name="ERR_ModuleInitializerMethodMustNotBeGeneric" xml:space="preserve">
+    <value>Module initializer method '{0}' must not have generic parameters</value>
+  </data>
+  <data name="ERR_ModuleInitializerMethodMustNotHaveParameters" xml:space="preserve">
+    <value>Module initializer method '{0}' must not have parameters</value>
+  </data>
+  <data name="ERR_ModuleInitializerMethodMustReturnVoid" xml:space="preserve">
+    <value>Module initializer method '{0}' must return void</value>
```

Comment:
>void [](start = 55, length = 4)

'void'?

Reply:
I didn't choose that because existing entries in this file used 'void' as a term rather than as inline code.

> Since '{0}' returns void, a return keyword must not be followed by an object expression

> The operation in question is undefined on void pointers

> '{0}': property or indexer cannot have void type

> __arglist cannot have an argument of void type

> The Conditional attribute is not valid on '{0}' because its return type is not void

> User-defined operators cannot return void

> Indexers cannot have void type

> Field cannot have void type

> Partial methods must have a void return type

etc

Reply:
> I didn't choose that because existing entries in this file used 'void' as a term rather than as inline code.

I think there are other examples, like:
```
  <data name="ERR_NoVoidParameter" xml:space="preserve">
    <value>Invalid parameter type 'void'</value>
  </data>
```

> Let me know what you'd like me to do.

I think using 'void' makes clear to the reader and especially to translators that they are dealing with a language concept. 

---
In reply to: [418178623](https://github.com/dotnet/roslyn/pull/43301#discussion_r418178623) [](ancestors = 418178623)